arm9/source/system/vram0.h

```diff
@@ -18,7 +18,7 @@
 #define VRAM0_EASTER_BIN       "easter.bin"
 
 
-#define VRAM0_OFFSET    0x080C0000
+#define VRAM0_OFFSET    0x080B7E00
```

Comment:
What's up with needing this?

Reply:
as commit said, to avoid bleeding into blacklisted range
or we can just shrink tar size limit 0x8200 bytes more to 223.5 KiB
I just prefer pretty-ish size over address plus it's already pretty small
(and we still have plenty space for actual arm9 binary)

Reply:
At that point I think we might as well just go with my other suggestion and embed the tar as an object in the ARM9 binary, rather than a section in FIRM.

0x080C0000 is already a pretty magical number, but 0x080B7E00 is even more so, and I'd rather avoid these.

Reply:
Well, then i guess just shrink the size even more for now, ppl need more will try to hack it themselves i think